.github/workflows/backend:check.yml

```diff
@@ -24,11 +24,24 @@ jobs:
           arguments: pmdMain pmdTest --parallel --continue
           build-root-directory: backend
           wrapper-directory: backend
-      - uses: actions/upload-artifact@v2
-        if: failure()
+      - name: Merge reports
+        run: |
+          sudo apt-get install zsh
+          head -n 5 backend/auth/auth-api/build/reports/pmd/main.xml > report.xml
+          zsh -c 'for file in backend/**/build/reports/pmd/main.xml;do;cat "$file" | tail -n +6 | sed "$ d" >> report.xml;done'
```

Comment:
za to se res rabi `zsh`?

Reply:
I couldn't get globstar (**) to work in bash. I would prefer if we could do that without `zsh` as installation takes ~30s.

Reply:
try bash -c `shopt -s globstar && ...`

Reply:
Will merge this as this PR makes it work and will try with bash in a follow up PR.